<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="bottomFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>811</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="tfPort">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>15020</string>
     </property>
    </widget>
    <widget class="QToolButton" name="sBtn">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>S</string>
     </property>
    </widget>
    <widget class="QToolButton" name="rightArrowBtn">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>100</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="tfAddress">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.1.106</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disconnectBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Déconnexion</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbWifibot">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CONTRÔLES WIFIBOT</string>
     </property>
    </widget>
    <widget class="QToolButton" name="upArrowBtn">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>60</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="zBtn">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connectBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Connexion</string>
     </property>
    </widget>
    <widget class="QToolButton" name="qBtn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Q</string>
     </property>
    </widget>
    <widget class="QToolButton" name="leftArrowBtn">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>100</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="downArrowBtn">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>140</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="dBtn">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>100</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>D</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbWifibot_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CONTRÔLES CAMÉRA</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbAddress">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse IP</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbPort">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QSlider" name="speedSlider">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>90</y>
       <width>41</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maximum">
      <number>240</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lbWifibot_3">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>VITESSE ROBOT</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbSpeed">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>70</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QWebEngineView" name="videoFrame" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>481</width>
      <height>351</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QFrame" name="capteursFrame">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>140</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbPos">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>POSITION DES CAPTEURS</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="pbBatterie">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
	text-align:center;
}

QProgressBar::chunk {
    background-color: #00FF00;
    width: 20px;

}</string>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="format">
      <string>%p%</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbBatterie">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>60</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Batterie : </string>
     </property>
    </widget>
    <widget class="QProgressBar" name="frontLeft">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
	text-align:center;
}

QProgressBar::chunk {
    background-color: #00FF00;
    width: 20px;

}</string>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QProgressBar" name="backLeft">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
	text-align:center;
}

QProgressBar::chunk {
    background-color: #00FF00;
    width: 20px;

}</string>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QProgressBar" name="frontRight">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
	text-align:center;
}

QProgressBar::chunk {
    background-color: #00FF00;
    width: 20px;

}</string>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QProgressBar" name="backRight">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
	text-align:center;
}

QProgressBar::chunk {
    background-color: #00FF00;
    width: 20px;

}</string>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="lbFront">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>AVANT</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbBack">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align:center;
</string>
     </property>
     <property name="text">
      <string>ARRIERE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="captureBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>131</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Capture d'écran</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
